eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/microfragments/eventdetails/fragments/views/ActionView.java

```diff
@@ -51,6 +52,13 @@ public ActionView(SchedjoulesViewEventDetailsActionBinding binding)
     }
 
 
+    public ActionView(SchedjoulesViewEventDetailsActionDirectionsBinding directionsBinding)
+    {
+        mRoot = directionsBinding.getRoot();
```

Comment:
better delegate to a private constructor like so
```
   this(directionsBinding.getRoot(), directionsBinding.schedjoulesEventDetailsDirectionsLabel);
```
Same goes for the other constructor of course.